bpdm-pool/src/test/kotlin/org/eclipse/tractusx/bpdm/pool/controller/MetadataControllerIT.kt

```diff
@@ -425,4 +432,30 @@ class MetadataControllerIT @Autowired constructor(
         )
         return rule
     }
+
+    @Test
+    fun `get all administrative-areas-level1`() {
+        initAdminAreasLevel1()
+        val page = poolClient.metadata.getAdminAreasLevel1(PaginationRequest())
+        assertThat(page.totalElements).isEqualTo(7L)
+        assertThat(page.content.size).isEqualTo(7)
+        val (countryCode, regionCode, regionName) = page.content.first { it.regionCode == "DE-BE" }
+        assertThat(countryCode).isEqualTo(DE)
+        assertThat(regionCode).isEqualTo("DE-BE")
+        assertThat(regionName).isEqualTo("Berlin")
+    }
+
+    private fun initAdminAreasLevel1() {
```

Comment:
I just noticed that when we clear our database between tests we also lose the regions we gained through the database migration. In my opinion the should assume that the region values are always in the database as they are based on an ISO norm. So it would be better exclude the table 'regions' from the database wipe and adapt the test to check that the endpoint returns the first page of iso norm regions

Reply:
Good idea!
This made me thinking that the regions should better be returned in a predictable order.
Now they are sorted by `countryCode` and `regionCode`.